collectors/mimecast/README.md

```diff
@@ -0,0 +1,91 @@
+# Mimecast collector
```

Comment:
Please add a section about how to configure mimecase credentials. And which Mimecast APIs are used.

Reply:
Also some links to mimecast docs will be appreciated.

Reply:
Done